duckling/language.py

```diff
@@ -20,3 +20,41 @@ class Language(object):
     UKRAINIAN = 'uk$core'
     VIETNAMESE = 'vi$core'
     CHINESE = 'zh$core'
+
+    SUPPORTED_LANGUAGES = [
```

Comment:
You could make this a set for a faster lookup (even though the difference should be minor).
Other than that it looks good! What do you think?